site/docs/develop/java.md

```diff
@@ -36,11 +36,11 @@ import org.projectnessie.client.api.NessieApiV1;
 import org.projectnessie.client.NessieClientBuilder;
 import org.projectnessie.model.Reference;
 
-NessieApiV2 api = NessieClientBuilder.builder()
-  .withUri(URI.create("http://localhost:19121/api/v2"))
-  .build(NessieApiV2.class);
+NessieApiV1 api = NessieClientBuilder.createClientBuilder(null, null)
+  .withUri(URI.create("http://localhost:19120/api/v1"))
+  .build(NessieApiV1.class);
 
-List<Reference> references = api.getAllReferences().get();
+List<Reference> references = api.getAllReferences().get().getReferences();
```

Comment:
It's much better to use `api.getAllReferences().stream()`, which leverages the paging functionality in REST v2.

Reply:
On next line, stream() example is given. So trying to understand what difference does it bring @snazy

Reply:
`stream()` will automatically paginate over all available entries, `get()` returns only the first page.